### ISSUE TITLE
After `init` a vite import fails

### ISSUE THREAD

**Issue by OddDev** [slidevjs/slidev#350]
**Describe the bug**
After the basic setup (`npm init slidev`), you get a blank page due to an `Internal Server Error`.
```
9:15:28 AM [vite] Internal server error: Failed to resolve import "@vueuse/shared" from "node_modules/.vite/@vueuse_motion.js?v=bb77a074". Does the file exist?
  Plugin: vite:import-analysis
  File: .../test/node_modules/.vite/@vueuse_motion.js?v=bb77a074
```

**To Reproduce**
1. Execute `npm init slidev`
2. Go through the setup
3. Execute `npm install`
4. Execute `npm run dev`
5. See the blank page
6. See the error in the console

**Desktop (please complete the following information):**
 - OS: Linux 5.13.0-7614-generic (Pop!_OS 21.04)
 - Browser: Firefox 92.0
 - Slidev version: v0.25.4
 - Node: v14.15.0
 - npm: 7.9.0

**Comment by OddDev** [comment]
```
9:15:28 AM [vite] Internal server error: Failed to resolve import "@vueuse/shared" from "node_modules/.vite/@vueuse_motion.js?v=bb77a074". Does the file exist?
  Plugin: vite:import-analysis
  File: /home/user/Test/test/node_modules/.vite/@vueuse_motion.js?v=bb77a074
  178|  
  179|  // node_modules/@vueuse/motion/dist/index.esm-bundler.js
  180|  import { tryOnUnmounted as tryOnUnmounted2, isFunction as isFunction2 } from "@vueuse/shared";
     |                                                                                ^
  181|  
  182|  // node_modules/popmotion/node_modules/tslib/tslib.es6.js
      at formatError (/home/user/Test/test/node_modules/vite/dist/node/chunks/dep-36bf480c.js:50163:46)
      at TransformContext.error (/home/user/Documents/Slidev/Test/test/node_modules/vite/dist/node/chunks/dep-36bf480c.js:50159:19)
      at normalizeUrl (/home/user/Test/test/node_modules/vite/dist/node/chunks/dep-36bf480c.js:75881:26)
      at async TransformContext.transform (/home/user/Documents/Slidev/Test/test/node_modules/vite/dist/node/chunks/dep-36bf480c.js:76016:57)
      at async Object.transform (/home/user/Test/test/node_modules/vite/dist/node/chunks/dep-36bf480c.js:50373:30)
      at async transformRequest (/home/user/Test/test/node_modules/vite/dist/node/chunks/dep-36bf480c.js:65414:29)
      at async viteTransformMiddleware (/home/user/Test/test/node_modules/vite/dist/node/chunks/dep-36bf480c.js:65542:32) (x3)
```

**Comment by activenode** [comment]
Could reproduce with the proposed way.

**Comment by Gholgot** [comment]
Hey @OddDev, here is a quick Fix I found :

In you `package.json` replace 
```json
 "dependencies": {
    "@slidev/cli": "^0.0.0-alpha.56",
    "@slidev/theme-default": "^0.0.0-alpha.56",
    "@slidev/theme-seriph": "^0.0.0-alpha.56"
  }
```
with 
```json
"dependencies": {
    "@slidev/cli": "0.25.4",
    "@slidev/theme-default": "0.21.0",
    "@slidev/theme-seriph": "0.21.0"
  }
```

Then run in your project's folder : `rm -rf node_modules && npm i`

**Comment by OddDev** [comment]
> Hey @OddDev, here is a quick Fix I found :
> 
> In you `package.json` replace
> 
> ```json
>  "dependencies": {
>     "@slidev/cli": "^0.0.0-alpha.56",
>     "@slidev/theme-default": "^0.0.0-alpha.56",
>     "@slidev/theme-seriph": "^0.0.0-alpha.56"
>   }
> ```
> 
> with
> 
> ```json
> "dependencies": {
>     "@slidev/cli": "0.25.4",
>     "@slidev/theme-default": "0.21.0",
>     "@slidev/theme-seriph": "0.21.0"
>   }
> ```
> 
> Then run in your project's folder : `rm -rf node_modules && npm i`

Hey, the right (or at least not the alpha version ones) dependencies are referenced in the [npm initializer template](https://github.com/slidevjs/slidev/blob/main/packages/create-app/template/package.json). And it's also executed correctly. Meaning my `package.json` looks like this:
```json
{
  "private": true,
  "scripts": {
    "build": "slidev build",
    "dev": "slidev --open",
    "export": "slidev export"
  },
  "dependencies": {
    "@slidev/cli": "^0.25.4",
    "@slidev/theme-default": "*",
    "@slidev/theme-seriph": "*"
  },
  "name": "test"
}
```

**Comment by Skarlso** [comment]
For the record, after installing `@vueuse/shared` with `npm install @vueuse/shared` I'm getting the following error:

```
[commonjs] Unexpected token (26:363439) in /Users/skarlso/goprojects/kube-cluster-sample/presentation/kube/node_modules/mermaid/dist/mermaid.min.js
```

**Comment by antfu** [comment]
Can you try `pnpm` or `yarn`? This is really weird to me as I can't actually reproduce it.

**Comment by OddDev** [comment]
> Can you try `pnpm` or `yarn`? This is really weird to me as I can't actually reproduce it.

Same with `pnpm`:
```
🦦pnpm init slidev
Need to install the following packages:
  create-slidev
Ok to proceed? (y) 

  ●■▲
  Slidev Creator  v0.25.8

✔ Project name: … slidev
  Scaffolding project in slidev ...
  Done.

✔ Install and start it now? … yes
✔ Choose the agent › npm

added 347 packages, and audited 348 packages in 30s

49 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

> dev
> slidev --open

Could not determine entry point from rollupOptions or html files. Skipping dependency pre-bundling.


  ●■▲
  Slidev  v0.25.8 

  theme   @slidev/theme-seriph
  entry   /home/user/Documents/Slidev/Test/slidev/slides.md

  slide show      > http://localhost:3030/
  presenter mode  > http://localhost:3030/presenter
  remote control  > pass --remote to enable

  shortcuts       > restart | open | edit


9:59:50 AM [vite] Internal server error: Failed to resolve import "@vueuse/shared" from "node_modules/.vite/@vueuse_motion.js?v=b04d2081". Does the file exist?
  Plugin: vite:import-analysis
  File: /home/user/Documents/Slidev/Test/slidev/node_modules/.vite/@vueuse_motion.js?v=b04d2081
  187|  
  188|  // node_modules/@vueuse/motion/dist/index.esm-bundler.js
  189|  import { tryOnUnmounted as tryOnUnmounted2, isFunction as isFunction2 } from "@vueuse/shared";
     |                                                                                ^
  190|  
  191|  // node_modules/popmotion/node_modules/tslib/tslib.es6.js
      at formatError (/home/user/Documents/Slidev/Test/slidev/node_modules/vite/dist/node/chunks/dep-24231913.js:50383:46)
      at TransformContext.error (/home/user/Documents/Slidev/Test/slidev/node_modules/vite/dist/node/chunks/dep-24231913.js:50379:19)
      at normalizeUrl (/home/user/Documents/Slidev/Test/slidev/node_modules/vite/dist/node/chunks/dep-24231913.js:75377:26)
      at async TransformContext.transform (/home/user/Documents/Slidev/Test/slidev/node_modules/vite/dist/node/chunks/dep-24231913.js:75517:57)
      at async Object.transform (/home/user/Documents/Slidev/Test/slidev/node_modules/vite/dist/node/chunks/dep-24231913.js:50590:30)
      at async doTransform (/home/user/Documents/Slidev/Test/slidev/node_modules/vite/dist/node/chunks/dep-24231913.js:64911:29)
```

**Comment by antfu** [comment]
Could anyone confirm if it fixes in v0.27.4? Thanks.
